store/sqlstore/store.go

```diff
@@ -152,6 +163,15 @@ func New(settings model.SqlSettings, metrics einterfaces.MetricsInterface) *SqlS
 
 	store.initConnection()
 
+	for _, conn := range store.GetAllConns() {
```

Comment:
Won't doing it only for master be sufficient?

Reply:
I think so it will be, i just didn't want to change too many things here. We currently are applying migrations for all connections, so I followed the same.

Reply:
Fair enough. Perhaps work to be done in a separate ticket?

Reply:
Amended on this PR, since we actually need a separate connection for migrations altogether.